test/functional/feature_notifications.py

```diff
@@ -36,54 +39,49 @@ def run_test(self):
         blocks = self.nodes[1].generate(block_count)
 
         # wait at most 10 seconds for expected file size before reading the content
```

Comment:
nit: comment should be updated: `...expected directory size...`

Reply:
also the comments below this one that refer to "file contents" should be "directory contents"